src/Components/Review.js

```diff
@@ -253,7 +255,7 @@ export default function Review({
             marginTop: "10px",
           }}
         >
-          <Tooltip title="Applaude this review" followCursor>
+          <Tooltip title={`Applaude this ${type}`} followCursor>
```

Comment:
Hmm, if `type` is not singular, would this end up being 'Applaud this comments'?  Same below for love and disagree buttons.

Reply:
Lolololol.  Great catch!